docs/getting-started/docker-quickstart.md

```diff
@@ -1,6 +1,20 @@
 # Docker Quickstart
 
-You can easily start experimenting with Tracee using the Docker image as follows:
+This section details how you can run Tracee through a container image.
+
+## Prerequisites
+
+!!! The container image currently does not support AArch64 architecture. Please ensure that you are on x64 system.
+
+Additionally, please ensure that Docker or another container runtime is working on your machine. 
+
+Note that we will provide support for multiple platforms, such as AArch64, in upcoming releases.
+In the meantime, have a look at [the Kubernetes guide](./kubernetes-quickstart).
```

Comment:
That is not 100% correct.
We actually support aarch64 architecture, and we have the aarch64 tag for that.
If one wants to use it, he can, and should replace latest tag with aarch64 tag.
In the future, we can try to support both architectures in the same docker image using a multi platform image https://docs.docker.com/build/building/multi-platform/